Casbin-OpenResty-Setup.md

```diff
@@ -0,0 +1,140 @@
+# Installing Lua-Cabsin on OpenResty
```

Comment:
I think this file content should be merged into the root README, as this repo is just for OpenResty. So there will be no `Casbin-OpenResty-Setup.md` file name any more.

Reply:
@hsluoyz Merged it into README.